src/browser/settings.jsx

```diff
@@ -33,6 +38,12 @@ var SettingsPage = React.createClass({
       showAddTeamForm: false
     });
 
+    appLauncher.isEnabled().then(function(enabled) {
```

Comment:
> Can't get/block the promise from node-auto-launch, so it only fills up way after we loaded everything for the settings menu, which then leads to a wrong checkbox.

You can use `didComponentMounted` API to reflect the initial state asynchronously instead of here. Would this be solution for your comment?